docs/components/AttachmentPreviewView.jsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+
+import Example, { CodeSample, ExampleCode } from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import { AttachmentPreview, Button } from "src";
+
+import "./AttachmentPreviewView.less";
+
+const cssClass = {
+  CONTAINER: "AttachmentPreviewView",
+  INTRO: "AttachmentPreviewView--intro",
+  PROPS: "AttachmentPreviewView--props",
+};
+
+export default class AttachmentPreviewView extends React.PureComponent {
+  static cssClass = cssClass;
+
+  state = {
+    showingPreview: false,
+  };
+
+  render() {
+    const { showingPreview } = this.state;
+
+    const file = {
+      id: "12345",
+      title: "Raccooooooon.jpg",
+      fileType: "jpg",
+      url:
+        "https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fppcorn.com%2Fus%2Fwp-content%2Fuploads%2Fsites%2F14%2F2016%2F01%2Fcute-raccoon-ppcorn.jpg&f=1&nofb=1",
+    };
```

Comment:
This is amazing, haha. How confident are we that it's not going to break at some point though, as opposed to something sadly far more boring (e.g. from the Clever branding page)?

Reply:
What if I upload it to s3, in assets.clever.com perhaps?

Reply:
I think that would be ideal, and in a way pretty straightforward! Not a blocker though, IMO.